Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:bionic
```

Comment:
Why did you move Dockerfile here?

Reply:
By mistake, I added Dockerfile and RBAC file sorry for  that!